src/html5/parser.rs

```diff
@@ -2145,8 +2147,25 @@ impl<'chars> Html5Parser<'chars> {
 
                 self.frameset_ok = false;
 
-                // Add attributes to body element
-                // @TODO add body attributes
+                let body_node_id = self.open_elements.iter().find(|node_id| {
```

Comment:
When we have multiple <body> tags, we need to copy the attributes from the body tags in the first / original body tag. The other body tags are further ignored